src/page-modules/departures/details/utils.ts

```diff
@@ -57,3 +57,9 @@ export function getSituationsToShowForCall(
       !alreadyShownSituationNumbers.includes(s.situationNumber),
   );
 }
+
+export function formatQuayName(quayName?: string, publicCode?: string | null) {
```

Comment:
Is this util function necessarily needed? Can't see that it brings a lot of benefit over just using the properties directly.

Reply:
Good point 🤔 I guess having an extra space at the end of the name doesn't really matter

Reply:
It shouldn't, as long as it doesn't affect any potential wrapping or anything, but it won't wrap mid word either as it is now.

Reply:
Actually, it can be `null` which would result in 
```ts
const name = 'Olav tryggvason';
const publicCode = null;
console.log(`${name} ${publicCode}`) // Olav tryggvason null
```

Reply:
`call.quay.publicCode ?? ''`

Reply:
True!

Reply:
Maybe easier to read the utils function? 😄

Reply:
No worries having it either.